iglu-server/README.md

```diff
@@ -0,0 +1,207 @@
+[su-exec][su-exec], as a sudo replacement, to run the Iglu Server as the non-root `snowplow` user.
+
+The container exposes the `/snowplow/config` volume to store the server configuration. If this
+folder is bind mounted then ownership will be changed to the `snowplow` user.
+
+The `-XX:+UnlockExperimentalVMOptions` and `-XX:+UseCGroupMemoryLimitForHeap` JVM options will be
+automatically provided when launching the server in order to make the JVM adhere to the memory
+limits imposed by Docker. For more information, see [this article][jvm-docker-article].
+
+Additional JVM options can be set through the `SP_JAVA_OPTS` environment variable.
+
+### Build
+
+1) Clone repository
+
+    `$ git clone git@github.com:snowplow/snowplow-docker.git`
+
+2) Navigate to `snowplow-docker/iglu-server/`
+
+    `$ cd snowplow-docker/iglu-server/`
+
+3) Build image    
+
+    `$ docker build -t iglu-server:0.x.0 0.x.0`
+
+    where `0.x.0` is version of Iglu Server one wants to run. Each version of Iglu Server has a dedicated folder under `iglu-server/` which contains a dedicated `Dockerfile`.
+
+### Usage
+
+After building an image for server, there are two options:
+
+  1) Running standalone image with postgres on docker host
+  2) Running with postgres in docker-compose
+
+Either way, a super API key needs to be inserted into database. `Iglu Server` makes use of an UUID key, called super API key to generate other API keys.
+
+After that, you can start using Iglu Server.
+
+### Run
+
+#### 1. Running standalone server image with postgres on docker host
+
+###### Important note
+
+Make sure that your config file contains proper db configuration and you've inserted your super API key.
+
+Running the container without arguments will print out its usage:
+
+```bash
+$ docker run iglu-server:0.x.0
+
+iglu-server 0.x.0
+Usage: iglu-server [options]
+
+  --help               Print this help message
+  --version            Print version info
+  --config <filename>  Path to custom config file. It is required and can not be empty.
+```
+
+Alternatively, we can mount a configuration folder and run the server:
+
+```bash
+$ docker run \
+  -d \
+  -v ${PWD}/config:/snowplow/config \
+  iglu-server:0.x.0 \
+  --config /snowplow/config/application.conf
+```
+
+If we want to specify additional JVM options, we can add the `SP_JAVA_OPTS` environment variable:
+
+```bash
+$ docker run \
+  -d \
+  -v ${PWD}/config:/snowplow/config \
+  -e 'SP_JAVA_OPTS=-Xms512m -Xmx512m' \
+  iglu-server:0.x.0 \
+  --config /snowplow/config/application.conf
+```
+
+#### 2. Running with postgres in docker-compose
+
+An example docker-compose for iglu server and postgres can be found at [the docker compose example][docker-compose-example].
+
+Before running, make sure to replace the API_KEY environment variable in `iglu-server/example/docker-compose.yml` with your preferred super API key. e.g. 1d9c7e70-012b-11e8-ba89-0ed5f89f718b
```

Comment:
I'm wondering if we should provide default key (this one for example). Removing one more manual step. Those who concerned about security will change it anyway. WDYT @alexanderdean, @oguzhanunlu, @BenFradet?

Reply:
yes this has already been discussed earlier in the reivew, c.f. snowplow/iglu#63

Reply:
Then I'll make use of this example key as default and put a reminder in README for people concerned about security.

Reply:
@BenFradet right, no need to provide that environment variable.